lib/wallet/nullclient.js

```diff
@@ -91,6 +92,16 @@ class NullClient extends EventEmitter {
     return { hash, height: 0, time: 0 };
   }
 
+  /**
```

Comment:
This JSDoc isn't correct. The blockhash isn't being used and `util.now` doesn't return a `Promise`

https://github.com/bcoin-org/bcoin/blob/2b2e53d83d4def08d25a2ed5c17c5638c3efa7b7/lib/utils/util.js#L58-L60

Reply:
Hmm... I think `nullclient.js` is only necessary as a stub for testing purposes only. The JSDoc is correct for the _actual_ usage.

Reply:
I think it's better to describe the actual behavior, vs what the stub is doing here. Ideally the stub would only exist in the tests. However, I think there would need to be further refactoring of the tests that is beyond the scope of this pull request.